### PR TITLE
🐛 Source Zendesk Support: Fix wrong types for schemas, add Transformer

### LINKED ISSUE #7441: Zendesk Support: Ticket custom fields not extracted properly after normalization
## Enviroment
- **Airbyte version**:  0.30.20-alpha
- **OS Version / Instance**: Amazon Linux / EC2 t3a.large
- **Deployment**: example are Docker or Kubernetes deploy env
- **Source Connector and version**: Zendesk Support 0.1.3
- **Destination Connector and version**: Bigquery 0.4.1
- **Severity**: Medium
- **Step where error happened**: When Look table result

## Current Behavior
in table `tickets` item in field `custom_fields` is `{"id":114101613833,"value":"87"}` however in table `tickets_custom_fields` the `value` field is `null`

## Expected Behavior
in table `tickets_custom_fields` the `value` field should be `87`

## Logs
*If applicable, please upload the logs from the failing operation. 
For sync jobs, you can download the full logs from the UI by going to the sync attempt page and 
clicking the download logs button at the top right of the logs display window.*

<details>
<summary>LOG</summary> 

```

replace this with
your long log
output here

```

</details>

## Image Reference

<details>
<summary>Images</summary> 

custom fields in tickets table

value in tickets table

value in tickets_custom_fields table

Screenshot from 2021-10-28 14-22-20

airbyte generated normalization query

Screenshot from 2021-10-28 14-22-51

</details>

## Steps to Reproduce
1. set catalog schema to `tickets`

## Are you willing to submit a PR?

Remove this with your answer.
Yes

### PR DESCRIPTION
**Pull request by yevhenii-ldv**
## What
*resolves #7441.
The problem was that the types were not specified for some of the fields in the schemas.*

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? If yes, please make sure to include it here and in any changelogs with the 🚨🚨 emoji
What is the end result perceived by the user?

#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py [hunk 1]
```diff
@@ -94,3 +95,39 @@ def test_comments_not_found_ticket(prepare_stream_args, status_code, expected_co
...
+        ),
+        (
+            {"id": 345, "custom_fields": [{"id": 5432123, "value": 55432.321}]},
+            {"id": 345, "custom_fields": [{"id": 5432123, "value": "55432.321"}]},
+        ),
+    ],
+)
+def test_transform_for_tickets_stream(prepare_stream_args, input_data, expected_data):
+    """Checks Transform in case when records come with invalid fields data types"""
+    stream = Tickets(**prepare_stream_args)
```
Comment (avida) [inline airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py:120]: What the point of testing it on stream level and manualy calling transorm method? Please test it on Source level like here connectors/source-iterable/unit_tests/test_source.py

**Review by avida — CHANGES_REQUESTED** [review]
Please update CDK version to 0.1.35 where false warning were fixed

File: airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py (hunk 1 above)
Comment (yevhenii-ldv) [reply airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py:120]: @avida I have not found a test in this file that would test Transformer.

**Review by Zirochkaa — APPROVED** [review]

**Review by avida — APPROVED** [review]

File: docs/integrations/sources/zendesk-support.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ We recommend creating a restricted, read-only key specifically for Airbyte acces
 
 | Version | Date | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| `0.1.5` | 2021-11-?? | [????](https://github.com/airbytehq/airbyte/pull/????) | Fix wrong types for schemas, add Transformer |
```
[the remark this answers is not in the document]
Comment (yevhenii-ldv) [reply docs/integrations/sources/zendesk-support.md:100]: I am going to request review from Airbyte team for this PR, and update the changeogs before publishing.

**Review by sherifnada — APPROVED** [review]

**Pull request merged**